CHANGELOG.md

```diff
@@ -1,16 +1,18 @@
 ## CHANGELOG
 
-### 3.6.0 (2021-MM-DD)
+### 3.X.X (YYYY-MM-DD)
 **SDK**
 - **Feature:** Added `getMiniAppInfoByPreviewCode` interface to get MiniAppInfo by preview code.
 - **Feature:** Added optional public key pinning through meta data.
 - **Feature:** Added additional rat events to track sdk feature usage.
+- **Feature:** Added signature verification process before downloading a MiniApp. HostApp also can enable the settings of SDK to verify signature. Please see user-guide.
```

Comment:
Could you also move the line about admob-latest from 3.5.0 to the 3.X.X section? 
`- **Feature:** Added `admob-latest` module to support latest admob sdk. `

Just so that we don't forget to do it later.